### ISSUE TITLE
[Bug]: Compatibility issue with Excalidraw plugin

### ISSUE THREAD

**Issue by mathisgauthey** [ObsidianPublisher/obsidian-github-publisher#315]
### Issue validation

- [X] I checked the issue to prevent duplicate
- [X] I checked my configurations files and the documentation
- [X] My issue concern doesn't concern the [Mkdocs template](https://github.com/ObsidianPublisher/template-main/)

### Plugin version

6.15.5

### Describe the bug

There's a compatibility issue with Excalidraw when activating Github-Publisher prevents from using any icons in the toolbar. It is weird.

[Link to the issue in the Excalidraw repo](https://github.com/zsviczian/obsidian-excalidraw-plugin/issues/1604)

### How to reproduce ?

1. Install Excalidraw
2. Install GIthub-Publisher
3. Try to create a drawing and click a try icon

### Minimal Reproducible Example

```markdown
None
```

### Configuration

```JSON
{
  "github": {
    "branch": "main",
    "automaticallyMergePR": true,
    "dryRun": {
      "enable": false,
      "folderName": "github-publisher"
    },
    "tokenPath": "%configDir%/plugins/%pluginID%/env",
    "api": {
      "tiersForApi": "Github Free/Pro/Team (default)",
      "hostname": ""
    },
    "workflow": {
      "commitMessage": "[PUBLISHER] Merge",
      "name": ""
    },
    "verifiedRepo": true
  },
  "upload": {
    "behavior": "fixed",
    "defaultName": "docs/posts",
    "rootFolder": "docs",
    "yamlFolderKey": "categories",
    "frontmatterTitle": {
      "enable": false,
      "key": "title"
    },
    "replaceTitle": [],
    "replacePath": [
      {
        "regex": "docs/blog",
        "replacement": "docs/blog/posts",
        "type": "path"
      }
    ],
    "autoclean": {
      "enable": true,
      "excluded": [
        "/^((?!docs\\/posts|docs\\/images).)*$/"
      ]
    },
    "folderNote": {
      "enable": false,
      "rename": "index.md",
      "addTitle": {
        "enable": false,
        "key": "title"
      }
    },
    "metadataExtractorPath": ""
  },
  "conversion": {
    "hardbreak": false,
    "dataview": true,
    "censorText": [
      {
        "entry": "/title:\\s\"([^\"]+)\"/i",
        "replace": "title: $1",
        "flags": "",
        "after": false
      },
      {
        "entry": "/(date_)(created: )(\\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}\\+\\d{2}:\\d{2})\\n(date_)(modified: )(\\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}\\+\\d{2}:\\d{2})/i",
        "replace": "date:\\n  $2$3\\n  updated: $6",
        "flags": "",
        "after": true
      }
    ],
    "tags": {
      "inline": false,
      "exclude": [],
      "fields": []
    },
    "links": {
      "internal": true,
      "unshared": false,
      "wiki": true,
      "slugify": "strict"
    }
  },
  "embed": {
    "attachments": true,
    "overrideAttachments": [],
    "keySendFile": [],
    "notes": false,
    "folder": "docs/images",
    "convertEmbedToLinks": "keep",
    "charConvert": "->",
    "useObsidianFolder": false
  },
  "plugin": {
    "shareKey": "share",
    "excludedFolder": [],
    "copyLink": {
      "enable": false,
      "links": "",
      "removePart": [],
      "transform": {
        "toUri": true,
        "slugify": "lower",
        "applyRegex": []
      }
    },
    "setFrontmatterKey": "Set"
  }
}
```

### Relevant log output

```bash session
Nope
```

### OS

Android, Windows

### Obsidian information

```bash session
SYSTEM INFO:
	Obsidian version: v1.5.3
	Installer version: v1.4.16
	Operating system: Windows 10 Pro 10.0.22631
	Login status: logged in
	Catalyst license: none
	Insider build toggle: off
	Live preview: on
	Base theme: adapt to system
	Community theme: none
	Snippets enabled: 4
	Restricted mode: off
	Plugins installed: 36
	Plugins enabled: 2
		1: Excalidraw v2.0.21
		2: Github Publisher v6.15.5

RECOMMENDATIONS:
	Custom theme and snippets: for cosmetic issues, please first try updating your theme and disabling your snippets. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community theme and snippets.
	Community plugins: for bugs, please first try updating all your plugins to latest. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community plugins.
```

**Comment by Lisandra-dev** [comment]
I don't think it's from my repo, as I don't touch the excalidraw plugin in any way?
Like, I just use the API from the official excalidraw…

**Comment by mathisgauthey** [comment]
> I don't think it's from my repo, as I don't touch the excalidraw plugin in any way?
> Like, I just use the API from the official excalidraw…

I was quite surprised as well, but using a new vault with only the two plugins, the issue is reproduced. Maybe some CSS issue ? That's weird, but I'd be glad to help.

**Comment by Lisandra-dev** [comment]
OIkay, I can reproduce it on a blank vault.

**Comment by Lisandra-dev** [comment]
Okay, I think it's because Publisher use an older version of Excalidraw :
With Publisher:
![image](https://github.com/ObsidianPublisher/obsidian-github-publisher/assets/30244939/88d08640-6a4f-45a0-9c53-da4a5ac49540)

Without:
![image](https://github.com/ObsidianPublisher/obsidian-github-publisher/assets/30244939/a41d78e1-6706-4df0-8012-dce07bd9c364)

**Comment by Lisandra-dev** [comment]
should be fixed in [f2e38a3](https://github.com/ObsidianPublisher/obsidian-github-publisher/commit/f2e38a3dd169754e55586f17b9a3f003d719a7b8)

Note: I load the export settings from the plugin settings directly, but more option can be used.

**Comment by mathisgauthey** [comment]
Yup, I confirm, this is now fixed. Thanks for being so quick to help like last time. Still enjoying your work, now using mkdocs for a much better user experience.
